e2e/index.ts

```diff
@@ -0,0 +1,14 @@
+export * from './inversify.config';
+export * from './inversify.types';
+export * from "./TestConstants";
+export * from "./pageobjects/login/ILoginPage"
+export * from "./driver/IDriver"
+export * from "./utils/DriverHelper"
+export * from "./pageobjects/dashboard/Dashboard"
+export * from "./utils/NameGenerator"
+export * from "./pageobjects/dashboard/NewWorkspace"
+export * from "./pageobjects/dashboard/workspace-details/WorkspaceDetailsPlugins"
+export * from "./pageobjects/ide/Ide"
+export * from "./pageobjects/ide/ProjectTree"
+export * from "./pageobjects/ide/Editor"
+export * from "./utils/workspace/TestWorkspaceUtil"
```

Comment:
Please add empty row

Reply:
And clarify please, what a purpose of adding this exports?

Reply:
These exports are needed for downstream tests to be able to reuse logic that is already implemented in upstream. You can see an example e.g. here https://github.com/redhat-developer/rh-che/pull/1438/files#diff-81af6d743c37c29cab3f9be6413a2365